bin/hipconfig

```diff
@@ -97,8 +108,15 @@ if (not defined $HIP_PLATFORM) {
     }
 }
 
-if ($HIP_PLATFORM eq "hcc") {
-    $CPP_CONFIG= " -D__HIP_PLATFORM_HCC__=   -I$HIP_PATH/include -I$HCC_HOME/include -I$HSA_PATH/include";
+if ($HIP_COMPILER eq "hcc") {
+    $CPP_CONFIG= " -D__HIP_PLATFORM_HCC__=  -I$HIP_PATH/include -I$HCC_HOME/include -I$HSA_PATH/include";
+}
+if ($HIP_COMPILER eq "clang") {
+    $HIP_CLANG_VERSION = `$HIP_CLANG_PATH/clang++ --version`;
+    $HIP_CLANG_VERSION=~/.*clang version ([^ ]+).*/;
+    $HIP_CLANG_VERSION=$1;
+
+    $CPP_CONFIG= " -D__HIP_PLATFORM_HCC__=  -I$HIP_PATH/include -I$HIP_CLANG_PATH/../lib/clang/$HIP_CLANG_VERSION/include -I$HSA_PATH/include";
```

Comment:
The clang include path is for the cuda wrappers. It needs to be -I$HIP_CLANG_PATH/../lib/clang to workaround libc++ depending on clang include path.

Reply:
Okay, so I don't need to give the 11.0.0/include directory?

Reply:
Thank you, just made this change.